command.go

```diff
@@ -331,11 +386,11 @@ func (c *Command) Help() error {
 
 // UsageString return usage string.
 func (c *Command) UsageString() string {
-	tmpOutput := c.output
+	tmpOutput := c.outWriter
 	bb := new(bytes.Buffer)
-	c.SetOutput(bb)
+	c.outWriter = bb
```

Comment:
It appears as if Usage could write to Stderr. Should we be setting `errWriter` as well?

Reply:
yes, to keep things backwards compatible this is necessary. I have improved this and added a new test